_posts/2019-05-06-gis-wisdom.md

```diff
@@ -77,20 +77,20 @@ Explore some of the [existing GIS open data](http://hub.arcgis.com/pages/open-da
 
 `refinement`
 
-The best datasets have lots of edits and editors.  They are continually being reviewed and refined. Think of multi-editing environments or Open Street Maps.  These datasets are so vast and valuable because they are 'live’ and diverse.  Everyone is bringing something unique to the product, which is difficult to produce with a small group of editors.
+The best datasets have lots of edits and editors. They are continually being reviewed and refined. Think of multi-editing environments or Open Street Maps. These datasets are so vast and valuable because they are "live" and diverse; everyone is bringing something unique to the product, which is difficult to produce with a small group of editors. It's the refinement process that makes the datasets more valuable. 
 
-But it's the refinement process that makes it valuable.  We too can continually refine through learning and sharing and practice and performance.  In turn, this advances our skills and adds to our value.
+We too can continually refine through learning and sharing and practice and performance. In turn, this advances our skills and adds to our value.
 
-Brush-up on some [essential editing vocabulary](http://desktop.arcgis.com/en/arcmap/10.3/manage-data/editing/essential-editing-vocabulary.htm).
+Brush up on some [essential editing vocabulary](http://desktop.arcgis.com/en/arcmap/10.3/manage-data/editing/essential-editing-vocabulary.htm).
 
 ### 6. Domains
 {: .text-left}
 
 `consistency`
 
-Domains are set up in the database to establish known consistency. They provide quality assurance on the front-end while ensuring consistency on the back-end.  What makes domains so great is that the end-user can reliably expect certain values or results.  Developers and cartographers benefit from this as well. 
+Domains are set up in a database to establish known consistency. They provide quality assurance on the front end while ensuring consistency on the back end. What makes domains so great is that they ensure that an end user can reliably expect certain values or results. Developers and cartographers especially benefit from this consistency. 
```

Comment:
Here I'm wondering about the "known consistency." Do we need the "known"? Or maybe we can explain it a bit more? Do you mean that domains establish consistency and make those standards of consistency visible to all users? Maybe I'm just overthinking this?

Reply:
let me ponder.  we can leave it for now